modules/core/04-channel/keeper/events.go

```diff
@@ -173,24 +163,15 @@ func emitRecvPacketEvent(ctx sdk.Context, packet types.Packet, channel types.Cha
 	})
 }
 
-// emitWriteAcknowledgementEvent emits an event that the relayer can query for
+// emitWriteAcknowledgementEvent emits an event for when an acknowledgement is written on the receiving chain.
 func emitWriteAcknowledgementEvent(ctx sdk.Context, packet types.Packet, channel types.Channel, acknowledgement []byte) {
 	ctx.EventManager().EmitEvents(sdk.Events{
 		sdk.NewEvent(
 			types.EventTypeWriteAck,
-			sdk.NewAttribute(types.AttributeKeyDataHex, hex.EncodeToString(packet.GetData())),
```

Comment:
we were emitting this value 4 times when receiving a packet (once in recv event, once in write ack then times two for ante handler execution). The packet data is provided in an arg to `MsgRecvPacket`